test/integration/annotation_spec.js

```diff
@@ -95,6 +95,31 @@ describe("Checkbox annotation", () => {
       );
     });
   });
+
+  fdescribe("issue15597.pdf", () => {
```

Comment:
Typo?

Reply:
No, oblivion (it's to run only this test, iirc f==focus)